pom.xml

```
@@ -11,7 +11,7 @@
     <packaging>jar</packaging>
     <groupId>org.webjars</groupId>
     <artifactId>pdf-js</artifactId>
-    <version>5.1.92-SNAPSHOT</version>
+    <version>5.2.133</version>
```

```suggestion
    <version>5.2.133-SNAPSHOT</version>
```
Maven does the actual release so we leave this as `-SNAPSHOT`